tools/Performance/DynamoPerformanceTests/PerformanceTestHelper.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System.Linq;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Validators;
+
+namespace DynamoPerformanceTests
+{
+    /// <summary>
+    /// Use this static class as Global helper to get all the prep data
```

Comment:
prep data?

Reply:
This helper used to get more preparation data, but I had to revert all the changes due to limitation of DotNetBenchmark. I will remove this